cypress/integration/styling_spec.js

```diff
@@ -4,4 +4,75 @@ describe('Styling', () => {
       .get('nav')
       .should('have.css', 'position', 'fixed')
     })
+
+    it("Make all the project images the same height", () => {
+      cy
+      .get('.project_image')
+      .should('have.css', 'height')
+      
+    })
+
+    it("Make all the project images the same width", () => {
+      cy
+      .get('.project_image')
+      .should('have.css', 'width')
+      
+    })
+
+    it("Give each project a solid black border of 2px", () => {
+      cy
+      .get('.project_card')
+      .should('have.css', 'border-style', 'solid')
+      .and('have.css', 'border-color', 'rgb(0, 0, 0)')
+      .and('have.css', 'border-width', '2px')
+    })
+
+    it("Give the project description a semi-transparent background with a color hex of #0c1427", () => {
+      cy
+      .get('.project_description')
+      .should('have.css', 'background-color', 'rgba(12, 20, 39, 0.3)')
+    })
+
+    it("Change the project description text to white", () => {
+      cy
+      .get('.project_description')
+      .should('have.css', 'color', 'rgb(255, 255, 255)')
+    })
+
+    it("Lay your project description over your project image using position absolute", () => {
+      cy
+      .get('.project_description')
+      .should('have.css', 'position', 'absolute')
+    })
+
+    it("Give you project description's parent element a relative position", () => {
+      cy
+      .get('.project_card')
+      .should('have.css', 'position', 'relative')
+    })
+
+    it("Fix your nav bar to the top using the z-index", () => {
+      cy
+      .get('nav')
+      .should('have.css', 'z-index', '100')
+    })
+
```

Comment:
A test is missing here: `Skills section should increment 10% in steps`. I think there is a normal mode and hard mode for solving this.

Normal mode: Find the `#languages` and `#libraries` elements and make sure they each have a `left` css property.

Hard mode: Similar to the z-index property solution, parse those left values into integers and make sure that they are greater than 0 and that the `left` attribute of `#libraries` is greater than the `left` attribute of `#languages`.